superset-frontend/src/dashboard/components/dnd/handleScroll.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+let scrollTopDashboardInterval: any;
+const SCROLL_STEP = 120;
+const INTERVAL_DELAY = 50;
+
+export default function handleScroll(dropPosition: string) {
```

Comment:
```suggestion
type DragPosition = 'DROP_TOP' | 'DROP_RIGHT' | ...;

export default function handleScroll(dropPosition: DragPosition) {
```

`dropPosition` could be of string literal type. But that could be in another refactoring PR when we migrate `superset-frontend/src/dashboard/util/getDropPosition.js` to TypeScript, too.